src/content/pages/volunteers.mdx

```diff
@@ -0,0 +1,120 @@
+* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us.
+
+# Perks for volunteers
+
+* Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-)
+* Be part of a great team and make new friends.
+* A special group photo with all volunteers (time will be announced on the #onsite-volunteers)
+
+# What Volunteering at EuroPython looks like
+
+As a part of EuroPython volunteers' fleet, you will be the face of the conference. Your prime directive will be to ensure everyone coming to the conference has a great time.
+We want volunteers to be welcoming, helpful, collaborative, and overall making sure everyone (including yourself) is comfortable.
+Despite the different tasks laid down below, volunteers represent a single unit. Sometimes this would mean that as a session chair you will be asked about where the quiet room is. We expect you to help this person out or point them to someone who can.
+
+We've put together an outline of some of the ways you can volunteer your time at the conference.
+
+## Registration Desk
+
+This is a very people-first position and will require spending a considerable time at the registration desk talking to attendees, along with carrying out the following activities:
+
+* Handing out registration badges
+* Checking student IDs
+* Answering general questions.
+
+We want to call out that volunteers at the registration desk will generally not be able to attend talks for the duration of the task.
+
+## T-Shirt Handout
+
+* Handing out the conference T-shirt
+* This job involves standing and lifting light items.
+
+We want to call out that this job is located at the registration desk, and you will generally not be able to attend talks for the duration of the task.
+
+## Room Manager
+
+A room manager is responsible for ensuring everything in a room runs smoothly for the entirety of a slot. Some of the activities a room manager is expected to carry out involve:
+
+* Insuring people are seated before a talk, and the room is not overcrowded
+* Making sure the session chair and speakers are in the room in time
+* Keeping in touch with the AV team to ensure everything is in order
+* Keeping the Code of Conduct (CoC) in mind and contact CoC committee members if needed
+
+We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.
+
+## Session Chairing
+
+A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:
+
+* Helping speakers in preparing their computers for the talks
+* Introducing the speakers to the audience
+* Signalling time left to the speakers
+* Making sure talks start and finish on time
+* Moderating Q&A session and help with passing around microphones and introduce remote attendees.
```

Comment:
```suggestion
* Moderating Q&A sessions and helping with passing around microphones and introducing remote attendees.
```

Reply:
This sentence needs some more work. Shouldn't "A and B and C" be written as "A, B, and C" ?

Reply:
Yeah I thought that the sentence has too many and. But I wasn't sure if introducing remote attendees was part of the "helping with". Perhaps it is clearer as "Moderating Q&A sessions, assisting with passing the microphones and introducing remote attendees."?
I was also not sure what "introducing remote attendees" exactly means but I didn't look into the meaning of these tasks much :)

Reply:
I think you just found a very valid point. (I did not think much either, just C&P from last year and edit where found).
Questions from remote are no longer introduced by the remote person but where read out from discord (if found).
What do you think about
"Moderating Q&A sessions, assisting with passing the microphones and read out questions from remote attendees."
?

Reply:
sounds good to me - that is if you are sure there will be remote questions as last year :)

Reply:
Updated as discussed.